packages/launcher/lib/darwin/util.ts

```diff
@@ -115,7 +115,7 @@ export async function darwinDetectionWorkaround (): Promise<FoundBrowser[]> {
   let args = ['./detection-workaround.js']
 
   if (process.env.CYPRESS_INTERNAL_ENV === 'development') {
-    args = ['-r', '@packages/ts/register.js'].concat(args)
+    args = ['-r', '@packages/ts/register.js'].concat(['./detection-workaround.ts'])
```

Comment:
`@packages/ts` isn't available in production builds, I don't think we can do this, have you tested that this still works in the built app on Big Sur?